src/span-data.ts

```diff
@@ -43,21 +42,12 @@ interface StackFrame {
   column_number?: number;
 }
 
-// Use 6 bytes of randomness only as JS numbers are doubles not 64-bit ints.
-const SPAN_ID_RANDOM_BYTES = 6;
-
-// Use the faster crypto.randomFillSync when available (Node 7+) falling back to
-// using crypto.randomBytes.
-const spanIdBuffer = Buffer.alloc(SPAN_ID_RANDOM_BYTES);
-const randomFillSync = crypto.randomFillSync;
-const randomBytes = crypto.randomBytes;
-const spanRandomBuffer = randomFillSync ?
-    () => randomFillSync(spanIdBuffer) :
-    () => randomBytes(SPAN_ID_RANDOM_BYTES);
+// This evaluates to a large whole number such that any fractional number
+// between 0 and 1 multiplied by it will always give a whole number.
```

Comment:
Interesting 🤔  do you have a source for this? I'm curious

Also, do you mean "any fractional number generated by `Math.random`"?

Reply:
So it has to do with the fact that the number is bigger than 1 / Number.EPSILON, but I think it's clearer to have a more meaningful maximum span ID. I changed it to just be the largest JS number that's less than 1^64-1, which is the maximum span ID as it's a 64-bit unsigned integer.